src/wp-includes/block-supports/layout.php

```diff
@@ -230,4 +250,65 @@ static function( $matches ) {
 	return $updated_content;
 }
 
-add_filter( 'render_block', 'wp_restore_group_inner_container', 10, 2 );
+add_filter( 'render_block_core/group', 'wp_restore_group_inner_container', 10, 2 );
```

Comment:
@Mamaduka Do these filter look okay to you?

In Gutenberg: https://github.com/WordPress/gutenberg/blob/689416bd5f10685fc7edffa4b2b1eb8aaa1f2951/lib/block-supports/layout.php#L243

Reply:
It's the correct filter. More context about the filter swap - https://github.com/WordPress/gutenberg/pull/38657#discussion_r822393700